### ISSUE TITLE
Proposed Analysis: investigate putative functionality of TP53 SV

### ISSUE THREAD

**Issue by kgaonkar6** [AlexsLemonade/OpenPBTA-analysis#953]
#### What are the scientific goals of the analysis?
We have investigated SNV  and CNV TP53 alterations in detail in `tp53-nf1-score` but there are other samples with high tp53 classifier scores but were not annotated via the 2 hit TP53 loss or activating mutation. 
![image](https://user-images.githubusercontent.com/34580719/110499453-40902300-80c6-11eb-84f2-753830b26046.png)

Thus in this [comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/837#issuecomment-790662695) @jharenza proposes checking SV breakpoints in/upstream of TP53 that might support the TP53 loss phenotype

#### What methods do you plan to use to accomplish the scientific goals?
Gather annotated manta calls to identify TP53 loss status :
- If SVs overlap TP53 gene  (should we include chr17p arm level loss ? ) or has breakpoint (bk) is within the gene

```
TAD = trans-activating domain (essential for function): AD1 (residues 1–42) and AD2 (residues 43–92)
DBD = DNA-binding domain (residues 102–292)
TD = tetramerization domain (residues 326–356)
```

Add `TP53 altered - loss` annotation if:
- SV overlaps or bk is within gene + any SNV (not adding CNV because it technically is a structural variation, thoughts?) ~ TP53 altered - loss
- SV overlaps or bk is within gene + cancer_predispositions == "Li-Fraumeni syndrome"  ~ TP53 altered - loss
- SV overlaps or bk is within gene + RNA-Seq > 0.5  ~ TP53 altered - loss

#### What input data are required for this analysis?
pbta-sv-manta.tsv.gz

#### How long do you expect is needed to complete the analysis? Will it be a multi-step analysis?
2 days

#### Who will complete the analysis (please add a GitHub handle here if relevant)?
@kgaonkar6 

#### What relevant scientific literature relates to this analysis?
@jharenza could you update the ticket if I missed anything and add references if possible?

Quick searchs got me:
http://dx.doi.org.proxy.library.upenn.edu/10.1038/nature25480
>Recurrently deleted regions were predominantly associated with epigenetic or cell cycle regulators, most commonly TP53, PTEN , SETD2 , and CDKN2A or CDKN2B 

https://doi.org/10.1093/jnen/nlaa124
> In conclusion, the clinical and molecular integrated analysis of this case showed that coexistence of chr 17p loss and chromothripsis in SHH medulloblastoma may indicate p53 pathway dysregulation and a “very high risk” tumor, even though the TP53 status is wild type.

**Comment by jharenza** [comment]
@kgaonkar6 thanks for adding this ticket! I think maybe we should keep it simple and just ask whether:

1. there is a breakpoint within the gene or covering the gene locus such that it is deleted and concurrently, there are either one or both of:
a - 0 copies of TP53 (we would already have captured them as loss)
b - expression less than 1 FPKM

**Comment by jharenza** [comment]
closed with #962
